File: packages/crisp/ios/Plugin/Classes/Events/SessionLoadedEvent.swift
import Foundation
import Capacitor

@objc public class SessionLoadedEvent: NSObject, Result {
    let sessionId: String

    init(sessionId: String) {
        self.sessionId = sessionId
    }

    @objc public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["sessionId"] = sessionId
        return result as AnyObject
    }
}

File: packages/crisp/android/src/main/java/io/capawesome/capacitorjs/plugins/crisp/classes/events/SessionLoadedEvent.java
package io.capawesome.capacitorjs.plugins.crisp.classes.events;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.crisp.interfaces.Result;

public class SessionLoadedEvent implements Result {

    @NonNull
    private final String sessionId;

    public SessionLoadedEvent(@NonNull String sessionId) {
        this.sessionId = sessionId;
    }

    @Override
    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("sessionId", sessionId);
        return result;
    }
}
